io/net/virt-net/network_virtualization.py

```diff
@@ -481,6 +482,7 @@ def test_rmdev_viosfailover(self):
         self.validate_vios_command('mkdev -l %s' % vnic_server, 'Available')
 
         networkinterface = NetworkInterface(device, self.local)
+        networkinterface.bring_up()
```

Comment:
@abdhaleegit I think after dlpar the interfaces should come_up automatically and we just need to verify their status whether they are up or not. why we bring them up?
@vaishnavibhat  ^^

Reply:
if you have sysfs static configuraiton.. the interface is expected to come up... 

But this is the case where I have dlpar test calling explicitly in a scenario cfg..so this is must to take care in that scenario